ask.py

```diff
@@ -953,11 +953,20 @@ def set_boilerplate():
 	flask_boilerplate += '\t\t\tif key in target.keys():\n'
 	flask_boilerplate += '\t\t\t\ttarget[key] = source[key]\n'
 	flask_boilerplate += '\n\t\treturn target\n'
-
+	
 	flask_boilerplate += '\n\t# Deprecated method\n'
 	flask_boilerplate += '\tdef quickSet(self, target, source):\n'
 	flask_boilerplate += '\t\treturn self.quick_set(target, source)\n'
 
+	flask_boilerplate += '\n\t@staticmethod\n'
+	flask_boilerplate += '\tdef require_keys(required_data_list, _body):\n'
+	flask_boilerplate += '\t\trequired_values_present = True\n'
+	flask_boilerplate += '\t\tfor element in required_data_list:\n'
+	flask_boilerplate += '\t\t\tif !(element in _body):\n'
```

Comment:
Chnage to `if element not in _body:`

Reply:
@Buscedv sure  no problem, and done.
Please if there's anything else tell me.